release-1.8/release_notes_draft.md

```diff
@@ -69,6 +69,9 @@ API surface for promotion.
  apps/v1beta2 group.
  - The pod.alpha.kubernetes.io/initialized has been removed.
 
+### Scheduling
+- Opaque Integer Resources (OIRs) are deprecated and may be removed in a
+  future release. Extended Resources (ERs) are a drop-in replacment.
```

Comment:
typo: replacement.

 Extended Resources (ERs) are a drop-in replacement for OIRs.